src/vivarium_nih_us_cvd/data/state_medication_data.csv

```diff
@@ -0,0 +1,121 @@
+,age_start,age_cat,sex,state,SBP_med_rate,LDL_med_rate,sbp_average,ldl_average,sbp_rr,ldl_rr,both_rr
```

Comment:
this is all dummy data but the real stuff should be the same columns and groupings